frontend/src/actions/prefAndProfileActions.tsx

```diff
@@ -131,7 +135,38 @@ export const updateUserPassword =
 
     const result = await operateUserDataRequest.updatePassword(payloadToSnakeCase)
     dispatch(changeUserPasswordSuccessfull(result.status))
-    toast.success(result.status)
+    toast.success(successMessage.successPasswordChange)
+  } catch (error) {
+    const destructuredError = {error}
+    const destructuredMessage = JSON.parse(destructuredError.error.message)
```

Comment:
No need to use `destructuredError`
`const parsedErrorMessage = JSON.parse(error.message)`
Why do we need `destructuredMessage`/`parsedErrorMessage`? I see that we only check that it exists and that's all. Will it work simply as:
```
if (error.message) {
dispatch(userPreferencesRequestFailed(error))
toast.error(error)
}
```
?

Reply:
if made simple as toast.error(error.message), it will show as - {errors: [{field: "detail", message: ["No active account found with the given credentials"]}]} - with too many extra brackets.